zkevm-circuits/src/table.rs

```diff
@@ -1809,9 +1809,11 @@ impl CopyTable {
                 real_length_left -= 1;
             }
         }
+        /* This does not work with negative test copy_circuit_invalid_tx_log
```

Comment:
not sure why work before.

Reply:
Because this test changes the first byte, which breaks a constraint (normal). Before this particular byte didn't break this particular constraint because of the mask which is different now.